app/models/hmpps_api/offender_base.rb

```diff
@@ -19,7 +19,7 @@ class OffenderBase
 
     attr_reader :crn,
                 :welsh_offender, :parole_review_date,
-                :ldu, :team
+                :ldu, :team, :responsibility
```

Comment:
Think this change isn't needed?

Reply:
@starswan Tests fail without this

Reply:
see comment further down - we shouldn't be exposing this object, we should be exposing responsibility_override? instead